user/pages/02.development/03.creating-a-skill/docs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Development Setup
+taxonomy:
+    category:
+        - docs
+---
+
+# Setting up the folders
+
+Once you've come up with your idea for a skill, make a new folder in `mycroft-core/mycroft/skills` with your desired skill name.
+
+For skills, there should generally be three things inside this folder - a folder called `dialog`, a file called `__init__.py`, and a folder called `vocab`. We'll go through these one by one so you know why each is there.
+
+###### Dialog
+The `dialog` folder contains subfolders for each spoken language the skill supports, each of which has `.dialog` files for what Mycroft should say when he executes a skill.
+
+For my welcome skill, I create the `dialog` folder. Inside, I create the subfolder `en-us`, which corresponds to American English. This folder should contain .dialog files, which will determine what Mycroft will say.
+For now, I create the file `Welcome.dialog`, which will contain various phrases that Mycroft will say. In it, I put on the first line `You're welcome`.
+###### Python script
+__init__.py is where you put the actual bulk of the skill, including a class that inherits from the MycroftSkill class and contains functions that pertain to how the skill operates. For now, I just create the file `__init__.py` and leave it blank.
+
+###### Vocab
+The `vocab` folder again contains subfolders for each langauge supported, each of which has `.voc` files that determine what Mycroft will listen for to trigger the skill.
+

```

Comment:
The `vocab` contains subfolders for each supported language like `en-us`. Inside each language folder we place `.voc` files which contains a set of keyword or phrases that Mycroft will listen for in order to trigger the skill.